src/System.Private.CoreLib/shared/System/Security/SecurityElement.cs

```diff
@@ -396,12 +397,12 @@ private static string GetEscapeSequence(char c)
             return c.ToString();
         }
 
-        public static string Escape(string str)
+        public static string? Escape(string? str)
```

Comment:
IsValidTag, IsValidText, IsValidAttributeName, and IsValidAttributeValue should all accept nullable.